### PR TITLE
Test Fix: Mariadb

### PR DESCRIPTION
**Pull request by mbfrahry**
```
--- PASS: TestAccMariaDbServer_createReplica (834.93s)
--- PASS: TestAccMariaDbConfiguration_logSlowAdminStatements (265.00s)
```

### CODE REVIEW

File: azurerm/internal/services/mariadb/mariadb_configuration_resource_test.go [hunk 1]
```diff
@@ -72,7 +72,7 @@ func TestAccMariaDbConfiguration_logSlowAdminStatements(t *testing.T) {
 		{
 			Config: r.logSlowAdminStatements(data),
 			Check: resource.ComposeTestCheckFunc(
-				data.CheckWithClient(checkValueIs("On")),
+				data.CheckWithClient(checkValueIs("on")),
```
Comment (katbyte) [inline azurerm/internal/services/mariadb/mariadb_configuration_resource_test.go:75]: Should we change the config to `On` to ensure this API respects case instead?

File: azurerm/internal/services/mariadb/mariadb_server_resource_test.go [hunk 2]
```diff
@@ -237,7 +237,6 @@ func TestAccMariaDbServer_createReplica(t *testing.T) {
 			Config: r.createReplica(data, version),
 			Check: resource.ComposeTestCheckFunc(
 				check.That(data.ResourceName).ExistsInAzure(r),
-				check.That("azurerm_mariadb_server").ExistsInAzure(r),
```
Comment (katbyte) [inline azurerm/internal/services/mariadb/mariadb_server_resource_test.go:240]: Should we instead check that the replica exists? `check.That("azurerm_mariadb_server.replica").ExistsInAzure(r),`

**Review by katbyte — CHANGES_REQUESTED** [review]
two comments left inline @mbfrahry

File: azurerm/internal/services/mariadb/mariadb_server_resource_test.go (hunk 2 above)
Comment (mbfrahry) [reply azurerm/internal/services/mariadb/mariadb_server_resource_test.go:240]: Ooo yes. Good call

**Review by katbyte — APPROVED** [review]
LGTM 👍

**Pull request merged**